marconi/marconi.cabal

```diff
@@ -0,0 +1,146 @@
+cabal-version: 3.0
+name:          marconi
+version:       0.1.0.0
+license:       Apache-2.0
+license-files:
+  LICENSE
+  NOTICE
+
+author:        Markus Läll
+maintainer:    markus.lall@iohk.io
+homepage:      https://github.com/input-output-hk/plutus-apps#readme
+bug-reports:   https://github.com/input-output-hk/plutus-apps/issues
+description:
+  Please see the README on GitHub at <https://github.com/input-output-hk/plutus-apps#readme>
+
+build-type:    Simple
+
+source-repository head
+  type:     git
+  location: https://github.com/input-output-hk/plutus-apps
+
+common lang
+  default-language:   Haskell2010
+  default-extensions:
+    DeriveFoldable
+    DeriveFunctor
+    DeriveGeneric
+    DeriveLift
+    DeriveTraversable
+    ExplicitForAll
+    GeneralizedNewtypeDeriving
+    ImportQualifiedPost
+    ScopedTypeVariables
+    StandaloneDeriving
+
+  ghc-options:
+    -Wall -Widentities -Wincomplete-record-updates
+    -Wincomplete-uni-patterns -Wmissing-import-lists
+    -Wnoncanonical-monad-instances -Wredundant-constraints
+    -Wunused-packages
+
+library
+  import:          lang
+  hs-source-dirs:  src
+  exposed-modules:
+    Marconi.Index.Datum
+    Marconi.Index.ScriptTx
+    Marconi.Index.Utxo
+
+  --------------------
+  -- Local components
+  --------------------
+  build-depends:
+    , plutus-ledger
```

Comment:
Create a ticket to remove this dependency, then add a TODO saying that it needs to be removed with the ticket link.

Reply:
Did it like this here 192f5cc1552eab8427c423b4415ee5c9c76ea533 because pre-commit hook re-formats the cabal file in such a way that the the todo comment is moved away into the title (near "Local components"). It also appears that comments at end of line are not permitted in cabal files.